Lib/importlib/_bootstrap_external.py

```diff
@@ -1092,8 +1092,17 @@ class ExtensionFileLoader(FileLoader, _LoaderBasics):
 
     """
 
-    def __init__(self, name, path):
-        self.name = name
+    def __init__(self, fullname=None, path=None, *, name=None):
+        if name is not None and fullname is not None:
+            raise TypeError("'name' and 'fullname' cannot be used "
+                            "simultaneously ('fullname' is preferred)")
+        elif name is not None:
+            _warnings.warn("the 'name' parameter is deprecated; use "
```

Comment:
Could you also update the list at https://docs.python.org/3.9/whatsnew/3.9.html#deprecated?